transcriptic/jupyter/objects.py

```diff
@@ -501,6 +512,147 @@ def _repr_html_(self):
                self.connection.get_route('view_data', data_id=self.id)
 
 
+class DataObject(object):
+    """
+    A DataObject holds a reference to the raw data, stored in S3, along with format and validation information
+
+    Attributes
+    ----------
+    id : str
+        DataObject id
+    dataset_id : str
+        Dataset id
+    data : bytes
+        Bytes fetched from the url
+    name: str
+        Dataset name
+    content_type: str
+        content type
+    format: str
+        format
+    size: int
+        size in bytes
+    status: string
+        valid vs invalid
+    url: str
+        download url which expires every 1hr.  Call `refresh` to renew
+    validation_errors: Array[str]
+        validation errors
+    container: Container
+        Container object that was used for this data object
+    aliquot: Aliquot
+        Aliquot object that was used for this data object
+    attributes: dict
+        Master attributes dictionary
+    connection: transcriptic.config.Connection
```

Comment:
Since we're not piping connection through in the `__init__`, this is no longer relevant